packages/core/src/Renderer.ts

```diff
@@ -350,6 +351,13 @@ export class Renderer extends Component implements IComponentCustomClone {
     );
   }
 
+  /**
+   * @internal
+   */
+  _isCulledByCamera(camera: Camera): boolean {
```

Comment:
This is just a comparison of `Layer`, not really  culled by camera.

Reply:
> This is just a comparison of `Layer`, not really culled by camera.

How about _isCulledByCameraCullingMask

Reply:
Has changed , named _isFilterByLayer

Reply:
done